x/oracle/types/ballot.go

```diff
@@ -1,12 +1,8 @@
 package types
 
 import (
-	"fmt"
-	"math"
-	"sort"
-	"strconv"
-
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"sort"
```

Comment:
```suggestion
	"sort"
	
	sdk "github.com/cosmos/cosmos-sdk/types"
```
apply import ordering convention